source/utils.js

```diff
@@ -4,6 +4,11 @@
  * @copyright Zemoga Inc
  */
 
+const PROTOCOL = 'https://';
```

Comment:
Should we set this constants in a config file to be imported instead of duplicate them in different files?